app/concepts/todo/operation.rb

```diff
@@ -0,0 +1,50 @@
+class Todo::Create < Trailblazer::Operation
+  include Model
+  model Todo, :create
+
+  contract do
+    property :title, validates: { presence: true }
+    property :description
+    property :todo_list_id
+    property :completed
+  end
+
+  def process(params)
+    # validate and save
+    validate(params[:todo]) do
+      contract.save
+    end
+  end
+
+end
+
+class Todo::Update < Trailblazer::Operation
+  include Model
+  model Todo, :update
+
+  contract do
+    property :title, validates: { presence: true }
+    property :description
+    property :completed
+  end
+
+  def process(params)
+    # validate and save
+    validate(params[:todo]) do
+      contract.save
+    end
+  end
+
+end
```

Comment:
That works. How we go one step further and `create` and `update` both inherit from a base class, there seems to be a more a "is a" relationship with a base class? That way we are just inheriting the shared `contract` and super of `process` each subclass could then have it's own extended logic potentially saving a refactor later as Service Objects have the potential for complex business logic/rules.